### PR TITLE
fix incorrect removeEventListener and refactor

### PR DESCRIPTION
**Pull request by ilyhryh**

### CODE REVIEW

File: press.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
     }
   }
 
   return window => {
-    window.addEventListener('keyup', keyUp)
-    window.addEventListener('click', click)
+      window.addEventListener('keyup', handleKeyUp)
+      window.addEventListener('click', handleClick)
     return () => {
-      window.removeEventListener('click', keyUp)
```
Comment (ilyhryh) [inline press.js:21]: bug is here

File: press.js [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-      pressed = event.target
+      event.target.classList.add(classes)
+      pressedElement = event.target
     }
   }
 
   return window => {
-    window.addEventListener('keyup', keyUp)
-    window.addEventListener('click', click)
+      window.addEventListener('keyup', handleKeyUp)
```
Comment (ai) [inline press.js:19]: Can you revert indent changes?

Comment (ilyhryh) [reply press.js]: Done, replace const with let for consistency

File: press.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
-    if (pressed) {
-      pressed.classList.remove(...pressedClass.split(' '))
+  function handleKeyUp() {
+    if (pressedElement) {
+        pressedElement.classList.remove(classes)
     }
   }
 
-  function click(event) {
+  function handleClick(event) {
```
Comment (ai) [inline press.js:11]: Can you revert name changes to keep consistency with other files

Comment (ilyhryh) [reply press.js]: Sure, I did this out of habit and for clarity

**Comment by ai** [comment]
Thanks. Released in 0.3.

**Pull request merged**